driver-mapping/src/main/java/com/datastax/driver/mapping/Mapper.java

```diff
@@ -932,19 +924,65 @@ public static Option saveNullFields(boolean enabled) {
             return new SaveNullFields(enabled);
         }
 
-        public Type getType() {
-            return this.type;
+        /**
+         * Creates a new Option object to specify whether an IF NOT EXISTS clause should be included in
+         * insert queries. This option is valid only for save operations.
+         * <p/>
+         * If this option is not specified, it defaults to {@code false} (IF NOT EXISTS statements are not used).
+         *
+         * @param enabled whether to include an IF NOT EXISTS clause in queries.
+         * @return the option.
+         */
+        public static Option ifNotExists(boolean enabled) {
+            return new IfNotExists(enabled);
+        }
+
+        final Type type;
+
+        protected Option(Type type) {
```

Comment:
Just noticed that I don't think `protected` is actually helpful here, I think it can go package-private. Since the other abstract methods on `Option` are package-private, I don't think we can expect anybody to `extends Option`, and so the only implementations possible are in this package